reloj-checador/config/database.yml

```diff
@@ -26,8 +26,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  username: postgres
-  password: 123456 #<-- which you have defiend while installing postgresql
+  username: frania
+  password: 12345 #<-- which you have defiend while installing postgresql
```

Comment:
it could be better to have a neutral username like `root`

Reply:
[https://github.com/Brightcoders-Bootcamps/kata-08-reloj-checador-team01-kata08/pull/16/commits/04f461c2e19281596bb07c5b184815dee4cc229d](add figaro gem for hidden params in the code)